src/data.json

```diff
@@ -5,27 +5,32 @@
       {
         "value": 100,
         "question": "When this is used, a variable can not be reassigned.",
-        "answer": "const"
+        "answer": "const",
+        "correct": null
```

Comment:
I'm not sure about keeping track of the correct answer here, because at one point in the future this will be in a database. Correct answers are session, not permanent data. Sync up with @sarahjting  to think about this problem.